de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/annotations/AnnotationManager.java

```diff
@@ -92,8 +96,20 @@ public void addSelectionAnnotation(
         AnnotationRange annotationRange;
 
         if (editor != null) {
-            RangeHighlighter rangeHighlighter = addRangeHighlighter(user, start,
-                end, editor, AnnotationType.SELECTION_ANNOTATION);
+            RangeHighlighter rangeHighlighter;
+
+            try {
+                rangeHighlighter = addRangeHighlighter(user, start, end, editor,
+                    AnnotationType.SELECTION_ANNOTATION);
+
+            } catch (IllegalArgumentException e) {
+                LOG.error(
+                    "The creation of a range highlighter for the file " + file
+                        + " with the bounds (" + start + ", " + end
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal " with the bounds (" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=203&issues=AWUe7ana8anAXINBEdcn&open=AWUe7ana8anAXINBEdcn)

[ISSUE_KEY]: AWUe7ana8anAXINBEdcn